packages/@okta/vuepress-site/docs/reference/rl-system-log-events/index.md

```diff
@@ -0,0 +1,263 @@
+| `operationRateLimitTimeUnit`       | String | Indicates the reset interval for `operationRateLmitTimeSpan` in minutes or seconds                                |
+| `operationRateLimitType`           | String | The type of rate limit event affected. Example types: `web_request`, `authenticator_otp_verification`, `sms_factor_enroll`, `event_hook_delivery`, `elastic_rate_limit_activated`, `phone_enrollment`, and so on|
+
+> **Note:** Additional information for some events may be included in the DebugContext object, such as for the Notification or Warning event types. For example:<br>
+> **For Notification event types**<br>
+> A preview-type event might contain a link to where you can toggle some behavior
+>
+> **For Warning event types**<br>
+> The event might include the threshold % that is being used to trigger the warning<br>
+>
+
+## DebugContext object examples
+
+The following is an example System Log rate limit event where too many enrollment attempts for the SMS factor were made.
+
+```json
+{
+  "actor": {
+    "id": "00uw8nGF9OiREtZyr0g3",
+    "type": "User",
+    "alternateId": "john.smith@example.com",
+    "displayName": "John Smith",
+    "detailEntry": null
+  },
+  "client": {
+    "userAgent": {
+      "rawUserAgent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10.15; rv:83.0) Gecko/20100101 Firefox/83.0",
+      "os": "Mac OS X",
+      "browser": "FIREFOX"
+    },
+    "zone": "null",
+    "device": "Computer",
+    "id": null,
+    "ipAddress": "127.0.0.1",
+    "geographicalContext": {
+      "city": null,
+      "state": null,
+      "country": null,
+      "postalCode": null,
+      "geolocation": {
+        "lat": 37.74,
+        "lon": -122.39999999999999
+      }
+    }
+  },
+  "authenticationContext": {
+    "authenticationProvider": null,
+    "credentialProvider": null,
+    "credentialType": null,
+    "issuer": null,
+    "interface": null,
+    "authenticationStep": 0,
+    "externalSessionId": "1025poeucCTQVK22GxJEK1Y-g"
+  },
+  "displayMessage": "Operation rate limit violation",
+  "eventType": "system.operation.rate_limit.violation",
+  "outcome": {
+    "result": "DENY",
+    "reason": "Too many enrollment attempts for SMS factor"
+  },
+  "published": "2020-12-10T04:46:12.033Z",
+  "securityContext": {
+    "asNumber": null,
+    "asOrg": null,
+    "isp": null,
+    "domain": null,
+    "isProxy": null
+  },
+  "severity": "WARN",
+  "debugContext": {
+    "debugData": {
+      "operationRateLimitSubtype": "SMS",
+      "operationRateLimitTimeUnit": "MINUTES",
+      "operationRateLimitScopeType": "user",
+      "operationRateLimitSecondsToReset": "99",
+      "operationRateLimitThreshold": "15",
+      "operationRateLimitTimeSpan": "5",
+      "requestUri": "/api/v1/authn/factors",
+      "url": "/api/v1/authn/factors?updatePhone=true",
+      "phoneNumber": "+14037894561",
```

Comment:
This looks like a real phone number - it comes from the example @zeusandrade-okta gave you.  We should swap in a fake phone number.